plugin/vimux.vim

```diff
@@ -70,12 +71,13 @@ function! VimuxOpenRunner()
   if _VimuxOption("g:VimuxUseNearest", 1) == 1 && nearestIndex != -1
     let g:VimuxRunnerIndex = nearestIndex
   else
+    let extraArguments = _VimuxOption("g:VimuxOpenExtraArgs", " ")
```

Comment:
```suggestion
    let extraArguments = _VimuxOption("g:VimuxOpenExtraArgs", "")
```

This isn't going to change the function at all, but it could someday be nice to test for an empty string an not have it be fooled be a default space.